images/win/scripts/Tests/Databases.Tests.ps1

```diff
@@ -5,4 +5,10 @@ Describe "MongoDB" {
     ) {
         "$ToolName --version" | Should -ReturnZeroExitCode
     }
-}
\ No newline at end of file
+}
+Describe "MySQL CLI" {
```

Comment:
Probably, we can name describe `MySQL` and It `MySQL CLI` so we will include some future tests here